caffe2/python/ideep/copy_op_test.py

```diff
@@ -63,3 +64,23 @@ def test_copy_from_ideep_fallthrough(self):
         workspace.RunOperatorOnce(op)
         X_ideep = workspace.FetchBlob("X")
         np.testing.assert_allclose(X, X_ideep)
+
+    def test_copy(self):
+        op = core.CreateOperator(
+            "Copy",
+            ["X"],
+            ["Y"],
+        )
+        op.device_option.CopyFrom(self._get_deep_device())
+        n = randint(1, 128)
+        c = randint(1, 64)
+        h = randint(1, 128)
+        w = randint(1, 128)
+        X = np.random.rand(n, c, h, w).astype(np.float32)
+        workspace.FeedBlob("X", X, self._get_deep_device())
+        workspace.RunOperatorOnce(op)
+        Y = workspace.FetchBlob("Y")
+        np.testing.assert_allclose(X, Y)
+
+if __name__ == "__main__":
```

Comment:
We don't need this.

Reply:
So,I need to remove the change of this file,is it right?

Reply:
Yes. Also please rebase with master and see if test failures will go away.